File: Tema 2 - Recogida de datos por teclado mediante formularios/Ejercicios/ejercicio7 Factura a partir de la base imponible/archives/7.js
function sumarJS() {
  const num1 = parseFloat(document.getElementById("num1").value);

  var solucion = `Base Imponible es = ${num1} Bs.
<br><br>Iva = ${(num1*0.1).toFixed(2)} Bs.
<br><br>Total a pagar = ${num1+(num1*0.1)} Bs.`;

  const resultadoDiv = document.getElementById("resultadoJavascript");
  resultadoDiv.innerHTML = `${solucion} `;
}

File: Tema 2 - Recogida de datos por teclado mediante formularios/Ejercicios/ejercicio7 Factura a partir de la base imponible/archives/7.py
import sys

# Obtener los argumentos de la línea de comandos
num1 = float(sys.argv[1])

print(f"""Base Imponible es = {num1} Bs.
<br><br>Iva = {(num1*0.1):.2f} Bs.
<br><br>Total a pagar = {num1+(num1*0.1)} Bs.""")
